Aplib.Core/Combinators.cs

```diff
@@ -0,0 +1,157 @@
+using Aplib.Core.Belief.BeliefSets;
+using Aplib.Core.Desire.Goals;
+using Aplib.Core.Desire.GoalStructures;
+using Aplib.Core.Intent.Actions;
+using Aplib.Core.Intent.Tactics;
+
+namespace Aplib.Core
+{
+    /// <summary>
+    /// Convenience class containing static methods for creating goal structures and tactics.
+    /// </summary>
+    public static class Combinators
+    {
+        #region GoalSructure combinators
```

Comment:
![image](https://github.com/team-zomsa/aplib.net/assets/35835627/2499e238-3c7f-40a0-ada1-6966b7ddcf9f)

```suggestion
        #region GoalStructure combinators
```